distribution/kpkginstall/runtest.sh

```diff
@@ -197,6 +197,29 @@ function copr_prepare()
   return 0
 }
 
+function download_install_package()
+{
+  $YUM install --downloadonly -y $1 | tee -a ${OUTPUTFILE}
```

Comment:
Can we have the stderr redirected here too, or does it cause trouble? It is useful for debugging to have the message logged directly

Reply:
np, added for both download & install